src/features/ITGovernance/Admin/Actions/RequestHome/index.tsx

```diff
@@ -0,0 +1,252 @@
+            {t('requestHome.sections.teamInfo.addAnother')}
+          </IconButton>
+        </Link>
+        <SystemIntakeContactsTable
+          contacts={data?.systemIntakeContacts?.allContacts}
+          loading={loading}
+          handleEditContact={contact => {
+            history.push({
+              pathname: `/it-governance/${systemIntake.id}/edit-point-of-contact`,
+              state: { contact }
+            });
+          }}
+          removeContact={handleRemoveContact}
+          pageSize={5}
+        />
+      </div>
+
+      {/* Request summary section */}
+      <div className="margin-y-4">
+        <h2 className="margin-bottom-4">
+          {t('requestHome.sections.requestSummary.heading')}
+        </h2>
+        <Grid row className="margin-bottom-4">
+          <div className="tablet:grid-col-6">
+            <p className="text-bold margin-top-0 margin-bottom-1">
+              {t('requestHome.sections.requestSummary.intakeRequestForm.title')}
+            </p>
+            <div className="margin-y-1">
+              <TaskStatusTag
+                status={systemIntake.itGovTaskStatuses.intakeFormStatus}
+              />
+            </div>
+            <Link to={`/it-governance/${systemIntake.id}/intake-request`}>
+              <IconButton
+                icon={<Icon.ArrowForward aria-hidden />}
+                type="button"
+                unstyled
+                iconPosition="after"
+              >
+                {t(
+                  'requestHome.sections.requestSummary.intakeRequestForm.view'
+                )}
+              </IconButton>
+            </Link>
+          </div>
+          <div className="tablet:grid-col-6">
+            <p className="text-bold margin-top-0 margin-bottom-1">
+              {t('requestHome.sections.requestSummary.businessCase.title')}
+            </p>
+            <div className="margin-y-1">
+              <TaskStatusTag status={bizCaseStatusToRender} />
+            </div>
+            <Link to={`/it-governance/${systemIntake.id}/business-case`}>
+              <IconButton
+                icon={<Icon.ArrowForward aria-hidden />}
+                type="button"
+                unstyled
+                iconPosition="after"
+              >
+                {t('requestHome.sections.requestSummary.businessCase.view')}
+              </IconButton>
+            </Link>
+          </div>
+        </Grid>
+
+        {/* Intake request form overview */}
+        <div
+          // Conditional rendering padding class to adjust padding becuase RichTextViewer adds a padding at the bottom
+          className={`bg-primary-lighter padding-3 ${expandedOverview ? 'padding-bottom-0' : ''}`}
+        >
+          <h3 className="margin-y-0">
+            {t('requestHome.sections.requestSummary.overview.heading')}
+          </h3>
+          <p className="easi-body-normal margin-y-0">
+            {t('requestHome.sections.requestSummary.overview.description')}
+          </p>
```

Comment:
This text also needs a larger top margin.

```suggestion
          <p className="easi-body-normal margin-bottom-0 margin-top-05">
            {t('requestHome.sections.requestSummary.overview.description')}
          </p>
```